### DISCUSSION TITLE
汉化工具说明

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/PokemonChineseTranslationRevise#23] [General]
# 汉化工具说明

https://blog.www.localhost/PokemonChineseTranslationRevise/Tools.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****pullDC 评论于 2020-11-24 16:35:27 (UTC+8)*****

码表文件在哪里啊

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Arceus 评论于 2020-12-02 01:41:03 (UTC+8)*****

求白金汉化码表

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Hung 评论于 2021-06-29 15:55:58 (UTC+8)*****

ndstool 缺失的DLL文件需要安装什么软件？
